.github/workflows/test.yml

```diff
@@ -0,0 +1,173 @@
+name: ovn-ci
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  GO_VERSION: 1.12
+  K8S_VERSION: v1.16.4
+  KIND_CLUSTER_NAME: ovn
+  KIND_INSTALL_INGRESS: true
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Free up disk space
+      run: sudo eatmydata apt-get remove --auto-remove -y aspnetcore-* dotnet-* libmono-* mono-* msbuild php-* php7* ghc-* zulu-*
+
+    - name: Set up Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ env.GO_VERSION }}
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+      
+    - name: Set up environment
+      run: |
+        export GOPATH=$(go env GOPATH)
+        echo "::set-env name=GOPATH::$GOPATH"
+        export PATH=$GOPATH/bin:$PATH
+        echo "::add-path::$GOPATH/bin"
+    
+    - name: Verify
+      run: |
+        pushd go-controller
+          make gofmt
+          make install.tools
+          make lint
+        popd
+
+    - name: Build
+      run: |
+        set -x
+        go get golang.org/x/tools/cmd/cover
+        pushd go-controller
+           make
+           make windows
+           COVERALLS=1 make check
+        popd
+        pushd dist/images
```

Comment:
@dave-tucker we could actually make the docker image verification a separate step (or maybe a separate job)? Doesn't need to happen now but something to think about. No real reason it's part of the build step except that it always has been in Travis.

Reply:
Opened #1214 to capture this for fixing later